react/tic-tac-toe/src/app/screens/Game/components/MatchHistory/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+
+import services from '~services/MatchesService'; // eslint-disable-line import/no-unresolved
+
+import styles from './styles.module.scss';
+
+class MatchHistory extends Component {
+  state = {
+    matches: []
+  }
+
+  componentDidMount() {
+    services.getMatches().then(response => this.setState({ matches: response.data }));
+  }
+
+
+  humanize = (property) => property.replace(/_/g, ' ')
+    .replace(/(\w+)/g, (match) => match.charAt(0).toUpperCase() + match.slice(1));
+
+  MATCH_RESULT_CLASS = {
+    player_one: 'winner-one', // eslint-disable-line camelcase
+    player_two: 'winner-two', // eslint-disable-line camelcase
+    tie: 'tie'
+  }
+
+  render () {
+    const { matches } = this.state;
+    return (
+      <table className={styles.matchHistory}>
+        <thead>
+          <tr className={styles.headerRow}>
+            <th className={styles.headerCell}>Player 1</th>
+            <th className={styles.headerCell}>Player 2</th>
+            <th className={styles.headerCell}>Winner</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          {matches.map(match => (
```

Comment:
If matches is `[]` you could display a message to the user that there is no matches.